bin/cli.js

```diff
@@ -6,10 +6,11 @@ const resourceGoal = require('../lib/goals/resource');
 const buildGoal = require('../lib/goals/build');
 const applyResources = require('../lib/apply-resources');
 const undeployGoal = require('../lib/goals/undeploy');
+let config;
```

Comment:
what's the purpose of moving this up, or is there still more code coming

Reply:
Oh rats. That's just left over from when I was debuglog'ing the config object in `cli.js`. I'll fix it.